api/proto/teleport/resourceusage/v1/account_usage_type.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.resourceusage.v1;
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/resourceusage/v1;resourceusagev1";
+
+// AccountUsageType is the type of the underlying account, either limitless or
+// limit-based.
```

Comment:
```suggestion
// AccountUsageType is the type of the underlying account, either unlimited or
// usage-based.
```

If it's one of those two, why do we need `ACCOUNT_USAGE_TYPE_UNSPECIFIED`?

Reply:
Just a best-practice with Proto enums https://protobuf.dev/programming-guides/style/#enums

Because every field is optional and will default to the zero-value. If the server forgets to set this field, getting "unspecified" might be more desirable than assuming e.g. "unlimited".

Reply:
What Justinas said. Also, it's only those two _today_, systems tend to evolve in unpredictable ways.